docker-compose.yml

```diff
@@ -0,0 +1,172 @@
+version: "3.9"
```

Comment:
I think this file should be in the build dir as well, but that is up for discussion.

Reply:
I see no problem at all 👌🏽